frontend/src/components/CommentsList/CommentsList.jsx

```diff
@@ -1,10 +1,10 @@
 import { UserComment } from 'components/Comments/UserComment/UserComment';
-import { Title } from 'components/Comments/UserCommentList/CommentList.styled';
+import { Title } from 'components/CommentsList/CommentsList.styled';
 import { useQuery } from 'react-query';
 
-export const CommentList = () => {
+const CommentsList = ({ announcementId }) => {
   const { data, isLoading, isError } = useQuery('commentList', async () => {
-    return await fetch('/commentLists').then((response) => response.json());
+    return await fetch(`/comments/${announcementId}`).then((response) => response.json());
   });
```

Comment:
Tego nie jestem pewien, ale sprawdzę i dam jutro znać :)))
Teraz jak patrzę na to cudo to zastanawiam się nad taką budową:

Pobrane głoszenie zawiera listę z IDs komentarzy.
My przekazujemy tablicę lub obiekt zawierający tablicę do Listy komentarzy, a ta lista komentarzy pyta poprzez tablicę promisów (z wykorzystaniem `Promise.all`) pod adres `/comments/${announcementId}`, Czyli wysyła np. 8 zapytań po jednym dla każdego zapytania, ale robi to w tablicy, czyli prawie w tym samym czasie.

Tak mi intuicja podpowiada, ale sprawdzę i dam znać rano

Reply:
Jak na discord dziś wysłałem